manual/vscode-extensions/vscode-icons/vscode-icons.nuspec

```diff
@@ -20,7 +20,7 @@
     <tags>microsoft visualstudiocode vscode icons extension foss cross-platform</tags>
     <releaseNotes>https://marketplace.visualstudio.com/items/robertohuertasm.vscode-icons/changelog</releaseNotes>
     <dependencies>
-      <dependency id="vscode" version="1.8.1" />
+      <dependency id="vscode" version="1.18.1" />
```

Comment:
Is it really necessary to update the version here now?
Isn't it just as well update it when we start using specific versions in the install script?

Reply:
If we change this package to use a specific version required Visual Studio Code version would be bumped to 1.30.0 (or 1.32.0 if they deliver the fix that extensions won't be auto updated). 

I'm OK to wait for the change to specific versions to have this fixed if we want to go this route

Reply:
I think at least eventually we'll want to go the route of using specific versions.
Maybe waiting a little, until most users had the chance to update to the latest vscode version (say a week or two at most).

Would you mind bumping the version of the package itself? Just so it can be pushed immediately when the PR is merged?

Reply:
If we go with specific version, we can close this one and I will migrate the package to an automated package which installs specific version